sdk/java/src/main/java/com/chain/http/Client.java

```diff
@@ -379,19 +379,24 @@ public void setProxy(Proxy proxy) {
   }
 
   private OkHttpClient buildHttpClient(Builder builder) {
-    OkHttpClient httpClient = new OkHttpClient();
+    OkHttpClient httpClient = builder.httpClient.clone();
+    httpClient.setFollowRedirects(false);
```

Comment:
Is this property carried across a `clone`? If so, it may be better to call this when the `httpClient` member is instantiated, back in the builder.